homeassistant/components/wled/light.py

```diff
@@ -164,6 +165,13 @@ async def async_preset(
         preset: int,
     ) -> None:
         """Set a WLED light to a saved preset."""
+        # The WLED preset service is replaced by a preset select entity
+        # and marked deprecated as of Home Assistant 2021.8
+        LOGGER.warning(
+            "The 'wled.preset' service is deprecated and replaced by a "
+            "dedicated preset select entity; Please use that entity to "
+            "change presets instead"
+        )
```

Comment:
Your call but I think it's always good to mention when the feature will be removed.

Reply:
Min 2 release cycles, could be 3 or 4 depends on time available.